staging/src/k8s.io/consts/admission/consts.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package admission
```

Comment:
Creating a new, parallel package structure doesn't seem like a net improvement.  What are you trying to fix?

Reply:
> hi @deads2k , you can see #59565

That is collecting into an existing package.  This creating a new one.  The two pulls are materially different.

Reply:
yeah, they are different, I don't confirm which scheme is better, so I commit another pr here.